docs-app/tests/acceptance/button-test.gts

```diff
@@ -1,16 +1,13 @@
 /* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
```

Comment:
I think I can also get rid of this now due to https://github.com/glimmerjs/glimmer-vm/pull/1406 being merged, but I'll test that out and put up a separate PR for those changes to keep these focused on the `--fix` path